src/HashBus.Viewer.MostMentioned/LeaderboardView.cs

```diff
@@ -124,23 +124,29 @@ static class LeaderboardView<TEntry> where TEntry : WebApi.IEntry
                     ColorConsole.WriteLine(new ColorToken[] { padding }.Concat(line).ToArray());
                 }
 
-                ColorConsole.WriteLine(
-                    padding,
-                    $"Total {itemsName}:".Gray(),
-                    " ",
-                    $"{currentLeaderboard?.Count ?? 0:N0}"
-                        .Color(currentLeaderboard?.Count - previousLeaderboard?.Count > 0 ? movementColors[-1] : movementColors[0]),
-                    $" since ".Gray(),
-                    $"{currentLeaderboard.Since?.ToLocalTime().ToString() ?? ("started")}".Gray());
+                if (currentLeaderboard.Since.HasValue)
+                {
+                    ColorConsole.WriteLine(
+                        padding,
+                        $"{currentLeaderboard.Since?.ToLocalTime().ToString()} to {currentLeaderboard.LastActivityDateTime?.ToLocalTime().ToString()}".Gray());
+                }
 
                 var maxMessageLength = 0;
                 var refreshTime = DateTime.UtcNow.AddMilliseconds(refreshInterval);
                 using (var timer = new Timer(c =>
                 {
                     var timeLeft = new TimeSpan(0, 0, 0, (int)Math.Round((refreshTime - DateTime.UtcNow).TotalSeconds));
-                    var message = $"\r{padding}{DateTime.UtcNow.ToLocalTime()} · Refreshing in {timeLeft.Humanize()}...";
-                    maxMessageLength = Math.Max(maxMessageLength, message.Length);
-                    ColorConsole.Write(message.PadRight(maxMessageLength).DarkGray());
+                    var tokens = new []
+                    {
+                        $"\r{padding}Total {itemsName}: ".DarkGray(),
+                        $"{currentLeaderboard?.Count ?? 0:N0}".Color(currentLeaderboard?.Count - previousLeaderboard?.Count > 0 ? movementColors[-1] : movementColors[0]),

```

Comment:
We can capture the result of

````C#
currentLeaderboard?.Count - previousLeaderboard?.Count > 0 ? movementColors[-1] : movementColors[0]
```

in a `movementColor` variable before the `using` statement. (I'd probably put this before the `maxMessageLength` initialiser.) This will shorten this line and also give us a wonderful micro-optimisation by not calculating it every second. :wink: